influence_benchmark/utils/wandb_logging.py

```diff
@@ -176,15 +201,20 @@ def log_iteration_data_to_wandb(turns_df, traj_df, iteration_step, top_n_trajs_p
             f"Avg influence: {env_avg_infl:.2f} ({env_stderr_infl:.2f})"
```

Comment:
I think it may be nice to also include the percentage stats here? Would require moving this whole print statement after the stats are computed